.gitlab-ci/env.sh

```diff
@@ -2,46 +2,60 @@
 
 #
 
-export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_NAME}
-export COMMIT=${COMMIT:=$CI_COMMIT_SHA}
+export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_SLUG}
 export COMMIT_TAG=${COMMIT_TAG:=$CI_COMMIT_TAG}
+export COMMIT=${COMMIT:=$CI_COMMIT_SHA}
+export ENVIRONMENT=${ENVIRONMENT:="dev.factory"};
+export HASH_SIZE=${HASH_SIZE:=7}
 export JOB_ID=${JOB_ID:=$CI_JOB_ID}
-export HASH_SIZE=${HASH_SIZE}
-export ENVIRONMENT=${ENVIRONMENT:="dev"};
-export CLUSTER_NAME="dev.factory";
-BRANCH_NAME_HASHED=$(printf "${BRANCH_NAME}" | sha1sum | cut -c1-${HASH_SIZE} )
+export PROJECT_PATH=${PROJECT_PATH:=$CI_PROJECT_PATH}
+
+BRANCH_NAME_HASHED=$( printf "${BRANCH_NAME}" | sha1sum | cut -c1-${HASH_SIZE} )
 export BRANCH_HASH=${BRANCH_HASH:=$BRANCH_NAME_HASHED}
 
+export DOMAIN="code-du-travail-numerique.${ENVIRONMENT}.social.gouv.fr";
+
+#
+
 #
 # For master branch we keep branch name as branch hash
 if [[ "${BRANCH_NAME}" = "master" ]]; then
   export BRANCH_HASH=master;
 fi
 
-# For versions we replace the version number v2.3.1 to v2-3-1
 if [[ -n "${COMMIT_TAG}" ]]; then
-  export ENVIRONMENT="incubateur";
-  export CLUSTER_NAME="incubateur"
+  # For versions we replace the version number v2.3.1 to v2-3-1
   export BRANCH_HASH=$( printf "${COMMIT_TAG}" | sed "s/\./-/g" );
 fi
 
 if [[ -n "${PRODUCTION+x}" ]]; then
-  export ENVIRONMENT="incubateur";
-  export CLUSTER_NAME="incubateur";
-  export BRANCH_HASH_DOT=""
-  export ELASTICSEARCH_HOST=elasticsearch:${ES_PORT};
+  export BRANCH_HASH=prod;
 else
-  export BRANCH_HASH_DOT="${BRANCH_HASH}."
-  export ELASTICSEARCH_HOST=elasticsearch-${BRANCH_HASH}:${ES_PORT};
+  export DOMAIN="${BRANCH_HASH}.${DOMAIN}";
```

Comment:
versions also need a prefix before the domain, no ?
`api.2-3-3.code-du-travail-numerique.incubateur.social.gouv.fr`

Reply:
yes see below prefixing